packages/forms/src/UIForm/fields/Text/Text.component.js

```diff
@@ -24,6 +24,7 @@ export default function Text(props) {
 			isValid={isValid}
 			label={title}
 			labelAfter
+			required={schema.required || false}
```

Comment:
`|| false` is unnecessary. FieldTemplate test the truthiness with `classnames()` to add a css class